ocp_resources/virtual_machine_clone.py

```diff
@@ -0,0 +1,87 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.utils import TimeoutSampler
+from ocp_resources.virtual_machine import VirtualMachine
+
+
+class VirtualMachineClone(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.CLONE_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        source_name=None,
+        source_type=None,
+        target_name=None,
+        label_filters=None,
+        annotation_filters=None,
+        new_mac_addresses=None,
+        new_smbios_serial=None,
+        yaml_file=None,
+        **kwargs,
+    ):
+        """
+        Create VirtualMachineClone object.
+
+        Args:
+            name (str): VirtualMachineClone name
+            source_name (str): the clone's source name
+            source_type (str): the clone's source type
```

Comment:
Maybe source_kind for here better?

Reply:
it's a bit confusing, but that's what we have in virtctl command help

$ virtctl create clone --help | grep source-type
      --source-type string              Specify the clone's source type. Default type is VM. Supported types: vm, vmsnapshot

Reply:
I get it, but this repo is not about `virtctl`, we call the API and in the API it is kind.

Reply:
done